### PR TITLE
Update copyrights: restore input file when script fails

### PR DESCRIPTION
**Pull request by ProditorMagnus**
```
Ravana@RavanaE480 /cygdrive/c/Users/Ravana/games/battle-for-wesnoth-1.19.2 $ ./update_copyrights.txt
fatal: not a git repository (or any parent up to mount point /cygdrive)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Traceback (most recent call last):
  File "./update_copyrights.txt", line 113, in <module>
    added.append(["", file_path, "", "", "", do_git(file_path), file_hash])
  File "./update_copyrights.txt", line 42, in do_git
    return str(check_output(["git", "log", "-1", "--format=%ad", "--date=format:%Y/%m/%d", file]), 'UTF-8').rstrip('\n')
  File "C:\Program Files (x86)\Python37-32\lib\subprocess.py", line 376, in check_output
    **kwargs).stdout
  File "C:\Program Files (x86)\Python37-32\lib\subprocess.py", line 468, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['git', 'log', '-1', '--format=%ad', '--date=format:%Y/%m/%d', 'data/campaigns/Legend_of_Wesmere/images/low-map.png']' returned non-zero exit status 128.

Ravana@RavanaE480 /cygdrive/c/Users/Ravana/games/battle-for-wesnoth-1.19.2 $ ./update_copyrights.txt
Traceback (most recent call last):
  File "./update_copyrights.txt", line 83, in <module>
    with open(options.input, encoding="utf-8") as csvfile:
FileNotFoundError: [Errno 2] No such file or directory: 'copyrights.csv'
```

### CODE REVIEW

**Comment by ProditorMagnus** [comment]
Might make sense to take next ~10 lines in block too

File: update_copyrights [hunk 1]
```diff
@@ -97,37 +97,46 @@ with open(options.input, encoding="utf-8") as csvfile:
...
+                elif csv_data[file_path][2].strip() == "" or csv_data[file_path][3].strip() == "":
+                    incomplete.append(csv_data[file_path])
+                elif not csv_data[file_path][2] in known_licenses:
+                    unknown_licenses.append(csv_data[file_path][2])
+                    incomplete.append(csv_data[file_path])
+                else:
+                    unchanged.append(csv_data[file_path])
+    if options.input == options.output:
+        os.remove(options.input + '-bak')
+except:
```
Comment (Wedge009) [inline update_copyrights:136]: Are there certain types of errors you're wanting to catch here? I think, generally, it's not a good idea to catch all errors.

Comment (ProditorMagnus) [reply update_copyrights:136]: When the error is rethrown it is ok to catch everything

**Comment by ProditorMagnus** [comment]
I think there is new implementation in progress so better to delay this pr.

**Comment by stevecotton** [comment]
Superseded by using exif data.

**Pull request closed without merging**